deepinv/transform/rotate.py

```diff
@@ -8,23 +10,31 @@ class Rotate(Transform):
     r"""
     2D Rotations.
 
-    Generates n_transf randomly rotated versions of 2D images with zero padding.
+    Generates n_trans randomly rotated versions of 2D images with zero padding.
+
+    See :class:`deepinv.transform.Transform` for further details and examples.
 
     :param degrees: images are rotated in the range of angles (-degrees, degrees)
```

Comment:
It seems to me that this is not what the function is doing, right?

It seems to be sampling from multiples of `360/degrees`?